docs/receipts.md

```diff
@@ -0,0 +1,42 @@
+# Document Generation in Indico
+
+Indico includes a module for generating documents, such as receipts and certificates, from templates that can be defined in the event or category. This feature is especially useful for conference organizers and administrators who need to provide attendees with customized documents quickly and efficiently.
+
+![](assets/document_templates/receipt-example.png)
+
+### Configuration
+
+Creating document templates requires knowledge of HTML and CSS and for security reasons is restricted to Indico admins. However, additional trusted users may be granted rights to create and edit templates
+In the _Document Templates_ section of the Indico administration page:
+
+![](assets/document_templates/configuring.png)
+
+### Creating a template
+
+Indico lets you define templates for various types of documents, including receipts, certificates, and more. These templates allow you to specify the layout, content, and placeholders for dynamic data using HTML with Jinja2 templates.
+
+Templates can be either created under an event, or a category, making them available for use in all of its events. These can be viewed, added, cloned and deleted from the management area under the _Customisation_ tab on events, or on its own _Document Templates_ tab on categories.
+
+![](assets/document_templates/edit.png)
+
+When creating a new template you must specify its name, and optionally define a default filename. Then you can design your template using the code editor. You can write HTML with Jinja2 templates, CSS and specify custom variables in YAML.
```

Comment:
Maybe also mention what the filename is for? We should also explain what at least the metadata.yml file is for and maybe a bit more about the placeholders